server/app.js

```diff
@@ -0,0 +1,3 @@
+import partyModel from './models/partyModel';
+
+console.log('partiex', partyModel.getAllParty);
```

Comment:
@frankly034 please remove the `console.log`

Reply:
Ok, I'm pushing an update with the changes you requested

Reply:
@Lumexralph I've pushed the changes you pointed out